src/frontend/magnify/packages/components/src/components/rooms/MyRooms/MyRooms.tsx

```diff
@@ -38,63 +38,69 @@ export const MyRooms = ({ baseJitsiUrl, rooms = [], ...props }: MyRoomsProps) =>
   const intl = useTranslations();
   const routing = useRouting();
   const isLog = KeycloakService.isLoggedIn();
+  const showClaimingRoom = !isLog && window.config.MAGNIFY_SHOW_REGISTER_LINK;
+
   return (
-    <CustomCard>
-      <Box align={'center'} direction={'row'} flex={true} justify={'between'}>
-        <Heading level={4}>
-          {intl.formatMessage(messages.myRoomCardTitle)}
-          {rooms?.length > 0 ? ` (${rooms?.length})` : ''}
-        </Heading>
-        <div>{isLog && <RegisterRoom />}</div>
-      </Box>
-      {isLog && (
-        <>
-          {props.isLoading && (
-            <Box align={'center'} height={'100px'} justify={'center'}>
-              <Spinner />
-            </Box>
+    <>
+      {(isLog || showClaimingRoom) && (
```

Comment:
In order to return early we could write something like that :

```ts
if (!showClaimingRoom) return null;

return (<CustomCard>...</CustomCard)
```

Reply:
Yes why not :)